[2024-11-29T06:07:16.023+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-29T06:07:16.031+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9868205040_2024-11-29T06-05-42-317794Z.make_api_call scheduled__2024-11-29T06:05:42.317794+00:00 [queued]>
[2024-11-29T06:07:16.034+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9868205040_2024-11-29T06-05-42-317794Z.make_api_call scheduled__2024-11-29T06:05:42.317794+00:00 [queued]>
[2024-11-29T06:07:16.034+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 3
[2024-11-29T06:07:16.039+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): make_api_call> on 2024-11-29 06:05:42.317794+00:00
[2024-11-29T06:07:16.044+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=173) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-29T06:07:16.044+0000] {standard_task_runner.py:72} INFO - Started process 174 to run task
[2024-11-29T06:07:16.045+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9868205040_2024-11-29T06-05-42-317794Z', 'make_api_call', 'scheduled__2024-11-29T06:05:42.317794+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/schedule_call.py', '--cfg-path', '/tmp/tmp77i4f28o']
[2024-11-29T06:07:16.046+0000] {standard_task_runner.py:105} INFO - Job 81: Subtask make_api_call
[2024-11-29T06:07:16.066+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9868205040_2024-11-29T06-05-42-317794Z.make_api_call scheduled__2024-11-29T06:05:42.317794+00:00 [running]> on host f4eb63bf5944
[2024-11-29T06:07:16.101+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9868205040_2024-11-29T06-05-42-317794Z' AIRFLOW_CTX_TASK_ID='make_api_call' AIRFLOW_CTX_EXECUTION_DATE='2024-11-29T06:05:42.317794+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-29T06:05:42.317794+00:00'
[2024-11-29T06:07:16.101+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-29T06:07:16.112+0000] {logging_mixin.py:190} INFO - API call to http://192.168.79.109:8001/call completed with status code 200
[2024-11-29T06:07:16.112+0000] {python.py:240} INFO - Done. Returned value was: 200
[2024-11-29T06:07:16.123+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-29T06:07:16.123+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9868205040_2024-11-29T06-05-42-317794Z, task_id=make_api_call, run_id=scheduled__2024-11-29T06:05:42.317794+00:00, execution_date=20241129T060542, start_date=20241129T060716, end_date=20241129T060716
[2024-11-29T06:07:16.137+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-29T06:07:16.148+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-29T06:07:16.149+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
